webapp/channels/src/components/widgets/advanced_textbox/advanced_textbox.tsx

```diff
@@ -102,7 +104,7 @@ const AdvancedTextbox = ({
     }
 
     return (
-        <div className='AdvancedTextbox'>
+        <div className={classNames('AdvancedTextbox', {classname})}>
```

Comment:
I may be misremembering how `classNames` work, but if you pass an object (in this case `{classname: 'some string'}`, it will just add the class `classname` if `some string` is truthy, which always is.

Shouldn't this be like this?
```suggestion
        <div className={classNames('AdvancedTextbox', classname)}>
```

Reply:
removed, not needed.